manifests/ipmi_exporter.pp

```diff
@@ -50,9 +52,10 @@
 # @param version
 #  The binary release version
 class prometheus::ipmi_exporter (
+  String[1] $config_file                  = '/etc/ipmi_exporter.yaml',
```

Comment:
can we use Stdlib:Absolutepath here?

Reply:
Done, I just originally opted for what was done with blackbox exporter, but using absolute path works too.